src/styles/pages/_live.scss

```diff
@@ -153,6 +153,28 @@
   }
 }
 
+web-tabs.w-event-tabs {
+  --primary-color: #{$WEB_PRIMARY_COLOR};
+  --hover-color: #{$GOOGLE_BLUE_50};
+  --active-color: #{$GOOGLE_BLUE_100};
+}
+
+table.w-event-schedule {
+  // TODO(robdodson): The generic table has `min-width: 512px`, which extends over the side of
```

Comment:
@samthor I think we usually enforce tables be wrapped in an outer `<div class="w-table-wrapper">` so they can overflow-x properly on small devices. But we can also see about removing the mind-width.

Reply:
Thanks for the clarification. In this case, there's no reason to let it extend—although cramped, the visual design works well at 320px.